common/ferrostar/src/routing_adapters/osrm/models.rs

```diff
@@ -500,4 +535,88 @@ mod tests {
             Some(vec!["right".to_string()])
         );
     }
+
+    #[test]
+    fn deserialize_incidents() {
```

Comment:
i got this example from hitting the api with points of -71.078874,42.376381 to -73.900821,40.659135 using the traffic routing profile.

the [documentation](https://docs.mapbox.com/api/navigation/directions/#route-leg-object) has a lot of details about the objects, but no sample unfortunately. i also found some things that came back from their api call that weren't in their documentation, so i intentionally left those out (ex length, affected_road_names_unknown, etc).